invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/DocumentsSearch/DocumentsSearch.js

```diff
@@ -176,6 +178,18 @@ export class DocumentsSearch extends Component {
   };
 
   render() {
+    const query = this.props.location.search;
+    const queryString = decodeURIComponent(query.match(/\?q=([^&]+)/)[1]);
```

Comment:
I would prefer we fix this in `react-searchkit` otherwise we will have to fix it in every search page, WDYT?